azurerm/data_source_kubernetes_cluster.go

```diff
@@ -612,6 +617,10 @@ func flattenKubernetesClusterDataSourceNetworkProfile(profile *containerservice.
 
 	values["network_plugin"] = profile.NetworkPlugin
 
+	if profile.NetworkPolicy != "" {
+		values["network_policy"] = profile.NetworkPolicy
```

Comment:
I think we should cast this to a string?
```suggestion
		values["network_policy"] = string(profile.NetworkPolicy)
```